packages/ketcher-react/src/script/ui/views/components/StructEditor/InfoPanel.tsx

```diff
@@ -135,16 +134,13 @@ const InfoPanel: FC<InfoPanelProps> = (props) => {
     >
       <StructRender
         struct={molecule}
-        id={groupName}
```

Comment:
Could you clarify, why have you removed these props?

Reply:
Since I replaced the class StructRender component with a functional component and updated all occurrences I also removed the redundant props: childRef was just set without any actual usage, the same for id, viewSz prop does not exists within the RenderOptions type. So I just cut them and tested it locally - struct renders works the same as before